pkgs/build-support/setup-hooks/move-systemd-user-units.sh

```diff
@@ -0,0 +1,25 @@
+#!/run/current-system/sw/bin/bash
```

Comment:
Is the shebang here necessary at all? Does it work in a sandboxed build or non-sandboxed build on non-NixOS?

Reply:
No, the shebang is not serving any function beside helping the editor and shellcheck detect that the code is expected to be Bash compatible.

I changed to a more generic `#!/usr/bin/env bash` line, though, that is probably more familar.